components/ws-manager/pkg/manager/internal/grpcpool/pool.go

```diff
@@ -6,49 +6,38 @@ package grpcpool
 
 import (
 	"fmt"
-	"strings"
 	"sync"
-	"time"
 
+	conn_pool "github.com/shimingyah/pool"
+	"golang.org/x/xerrors"
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/connectivity"
 )
 
 var (
 	// ErrPoolClosed is returned if Get is called after Close
 	ErrPoolClosed = fmt.Errorf("pool is closed")
+	// ErrNoWSDeamonFoundInPool is returned is there is no ws-daemon pod in the host
+	ErrNoWSDeamonFoundInPool = fmt.Errorf("no ws-daemon found in the host")
 )
 
-// Factory is a function which creates new grpc connections
-type Factory func(host string) (*grpc.ClientConn, error)
+// PoolOptions is a function which creates new grpc connections
```

Comment:
Should it be `// Dial is a function which creates new grpc connections`?

Reply:
fixed